ext/mixed/js/api.js

```diff
@@ -65,10 +65,6 @@ function apiNoteView(noteId) {
     return _apiInvoke('noteView', {noteId});
 }
 
-function apiTemplateRender(template, data, dynamic) {
-    return _apiInvoke('templateRender', {data, template, dynamic});
-}
-
```

Comment:
This causes template rendering on the settings page to stop working. It should probably be switched to use the `bg` version of `api.js`, but that could require adding some missing functions to it.

Reply:
Reverted in 61c6a753ce964700e3ef9f6db6bb1137204caf46. I forgot that the bg version of api.js is only there for the background.html page. It would be optimal to remove that one at some point, but that will take some more evaluation.